### ISSUE TITLE
Can't scroll on addon pages of the chrome web store

### ISSUE THREAD

**Issue by glacambre** [tridactyl/tridactyl#1889]
Tridactyl version: 1.17.0pre3184-691b830
Firefox version: Mozilla Firefox 69.0.2
Operating system: linux

Reproducer:
`tabopen https://chrome.google.com/webstore/detail/firenvim/egpjdkipkomnmjhjmdamaniclmdlobbo?hl=en`

Oddly not reproducible if you're on the web store's main page and navigate there by using the search box + hints. This probably means that the page manages to steal Tridactyl's focus when you open it directly.

**Comment by bovine3dom** [comment]
Seems to scroll fine for me now

**Comment by glacambre** [comment]
It doesn't for me but I'm unlikely to ever attempt to figure out why, so I'm fine with this being closed 🤷.

**Comment by pataquets** [comment]
Doing some issue gardening here. Works for me :tm:. @glacambre: I guess the issue can be closed.

Tried it, and _it ended up_ scrolling fine, but previously I had to...
1) close the "switch to Chrome" site popup:
![image](https://github.com/tridactyl/tridactyl/assets/1286254/7bf10a46-844c-4d2e-9db9-199c4d5e4b41)
2) move the focus away from the search box (evidence of "me stubbornly trying until I realized it" are left for completeness and enjoyment :smile:):
![image](https://github.com/tridactyl/tridactyl/assets/1286254/cf9b73d8-170f-4598-bc68-4a705705b912)

Not meaning it was @glacambre's problem back in the day, just warning everybody else FAQ-style as sort of UX feedback.
